IDEAS/Templates/Interfaces/Building.mo

```diff
@@ -8,7 +8,8 @@ model Building
   replaceable package MediumAir=IDEAS.Media.Air;
 
   parameter Boolean standAlone=true;
-  parameter Boolean isDH=false "True if the building is connected to a DH grid";
+  // parameter isDH defining if the building is connected to a DH grid is now set in the heating system model
```

Comment:
I think this comment can be removed since it explains nothing in the code (it explains why something used to be in the code) and therefore it can be confusing?

Reply:
Indeed, I was just thinking of the poor people suddenly having errors in their models that previously worked, and not knowing why (speaking from personal experience). But it's better I put these in the review history (hadn't even thought about it).